pytorch_lightning/accelerators/accelerator.py

```diff
@@ -133,11 +112,6 @@ def lightning_module(self) -> "pl.LightningModule":
         """
         return self.training_type_plugin.lightning_module
 
-    @property
```

Comment:
Why do we need to move root device in this PR?

Reply:
From the PR title: `move optimizer related logics`. I believe this could be within another PR.

Reply:
@four4fish It would be fine to call l`ightning_model.trainer.accelerator.root_device` temporarily for this PR.

Reply:
Sure, will update then!

Reply:
updated